### ISSUE TITLE
Get-DbaErrorLogFile

### ISSUE THREAD

**Issue by pollusb** [dataplat/dbatools#9333]
### Summarize Command's Functionality

Get-DbaErrorLog does not provide access to errorlog info when the SQL instance is down. This function would return the files (or the content) to diagnose a problem that might have to do with the fact that it's down.

### Is there a command that is similiar or close to what you are looking for?

No

**Comment by niphlod** [comment]
I don't think we can accomodate this as explained in #9330, the api is wildly different.
That being said, if the instance is offline, isn't Get-DbaWindowsLog a more valuable candidate ?
If not, the only possible way to get at least the path would be Get-DbaStartupParameter, then hoping that it points to a path where the errorlogs are.

**Comment by potatoqualitee** [comment]
@claude what do you think? close or niph's suggestion?

**Comment by pollusb** [comment]
I also agree. Let's close this !
